src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -1235,6 +1235,33 @@ private static double SafeNextDouble(this IServiceProvider services)
             return random.SafeNextDouble();
         }
 
+        private static decimal SafeNextDecimal(this IRandomService random)

```

Comment:
>SafeNextDecimal [](http://example.com/codeflow?start=31&length=15)

super nit:
can you move this to its own class 

internal static class RandomServiceExtensions 
{
public static decimal SafeNextDecimal(this IRandomService random)
}

That can live in same file as IRandomService, or a new file.

Reply:
updated